### PR TITLE
Version upgrade 2.12

### PR DESCRIPTION
**Pull request by edmundnoble**
* [x] Double check CHANGELOG.md
    * [ ] Should we include any breaking pact changes in more detail?
* [x] Double check next service date

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+
+*   The default setting for pruning the chainweb database has been changed to 
+    "none" to reduce unnecessary work and speed up node start times. (#1332)
+
+*   Nodes will now take a snapshot of the chainweb database (RocksDB) if sent 
+    the SIGUSR1 signal. These snapshots can be used for backups and do not take
+    significant space until the database has diverged significantly from the 
+    snapshot. Also, SIGUSR2 no longer terminates the node. (#1328)
+
+*   Upgrade Pact version to 4.2.0. (#1323)
```
Comment (larskuhtz) [inline CHANGELOG.md:21]: Maybe move this to the top of the list, since it is the most important change?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
 # `chainweb-node` Changelog
 
+## 2.12 (2022-01-07)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2022-01-13T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
+
+This version will stop working on **2022-02-24T00:00:00Z**.
```
Comment (larskuhtz) [inline CHANGELOG.md:9]: fyi, this is 6 weeks

File: chainweb.cabal [hunk 3]
```diff
@@ -10,7 +10,7 @@ license:      BSD-3-Clause
 license-file: LICENSE
 author:       Chainweb Dev Team
 maintainer:   chainweb-dev@kadena.io
-copyright:    Copyright (C) 2018 - 2021 Kadena LLC
+copyright:    Copyright (C) 2018 - 2022 Kadena LLC
```
Comment (larskuhtz) [inline chainweb.cabal:13]: We should also update the copy right date in the LICENSE file.

**Review by larskuhtz — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (mightybyte) [reply CHANGELOG.md:9]: Confirmed

File: node/ChainwebNode.hs [hunk 4]
```diff
@@ -473,10 +473,10 @@ pkgInfoScopes =
 -- -------------------------------------------------------------------------- --
 -- main
 
--- SERVICE DATE for version 2.11
+-- SERVICE DATE for version 2.12
 --
 serviceDate :: Maybe String
-serviceDate = Just "2022-01-13T00:00:00Z"
+serviceDate = Just "2022-02-24T00:00:00Z"
```
Comment (mightybyte) [inline node/ChainwebNode.hs:479]: Confirmed.  Also...this matches the date in the changelog.

**Review by larskuhtz — APPROVED** [review]

**Review by sirlensalot — APPROVED** [review]

**Pull request merged**